src/researchhub_document/related_models/researchhub_unified_document_model.py

```diff
@@ -1,23 +1,352 @@
+from datetime import datetime, timedelta
```

Comment:
@le-sun , Can you hold a 15-min chat to go over how the filters are working now. 
Would be good to hear the updated architecture for the entire team.

Reply:
@calvinhlee23 We can go over the filter changes after the sync tomorrow